paddle/fluid/API.spec

```diff
@@ -318,6 +318,7 @@ paddle.fluid.layers.iou_similarity ArgSpec(args=['x', 'y', 'name'], varargs=None
 paddle.fluid.layers.box_coder ArgSpec(args=['prior_box', 'prior_box_var', 'target_box', 'code_type', 'box_normalized', 'name'], varargs=None, keywords=None, defaults=('encode_center_size', True, None))
 paddle.fluid.layers.polygon_box_transform ArgSpec(args=['input', 'name'], varargs=None, keywords=None, defaults=(None,))
 paddle.fluid.layers.yolov3_loss ArgSpec(args=['x', 'gtbox', 'gtlabel', 'anchors', 'class_num', 'ignore_thresh', 'loss_weight_xy', 'loss_weight_wh', 'loss_weight_conf_target', 'loss_weight_conf_notarget', 'loss_weight_class', 'name'], varargs=None, keywords=None, defaults=(None, None, None, None, None, None))
+paddle.fluid.layers.box_clip ArgSpec(args=['input', 'im_info', 'inplace', 'name'], varargs=None, keywords=None, defaults=(False, None))
```

Comment:
try to avoid `inpace` argument and register the operator as inplace? @dzhwinter can help you

Reply:
avoid provide an API with inplace, because it maye caused bug when the inplaced var used by more than one op.
, we can register an inplace handle like 
https://github.com/PaddlePaddle/Paddle/pull/14975/files#diff-279b905e23844ad111be46fd44ceb6dbR605
Our framework will automatically make the variable inplace-reused.

Reply:
Removed inplace argument in api. When https://github.com/PaddlePaddle/Paddle/pull/14975/files merges, I'll add add inplace registration in next pr